client/state/selectors/test/get-blog-stickers.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getBlogStickers } from '../';
+
+describe( 'getBlogStickers()', () => {
+	it( 'should return stickers for a known blog', () => {
+		const stateIn = {
```

Comment:
I've got no strong prefs - I was following another set of conventions :) Would be good to standardise across Calypso.

I quite like 'stateBefore'/'stateAfter' perhaps? Gone with yours though 👍